### ISSUE TITLE
unable to use custom Mail Message class

### ISSUE THREAD

**Issue by yaroslav-zenin** [dotmailer/dotmailer-magento2-extension#522]
I've created own `\Namespace\Mail\Message` implement `\Magento\Framework\Mail\MessageInterface` 
but unable to use it because of signature
```php
<?php

namespace Dotdigitalgroup\Email\Plugin;

class MessagePlugin
{
    public function afterSetBody(\Magento\Framework\Mail\Message $message, $body)
   {}

   // should be 
    public function afterSetBody(\Magento\Framework\Mail\MessageInterface $message, $body)
    {}
}
```

also it will be strictly required on Magento 2.3 because they didn't extends from \Zend\Mail anymore

**Comment by simon-letch** [comment]
Hi @yaroslav-zenin,

We have a fix to the method signature going through code review and UAT now.

Thanks,
Simon

**Comment by yaroslav-zenin** [comment]
@simon-letch thanks
look forward for the new release ;)

**Comment by simon-letch** [comment]
Hey @yaroslav-zenin,

As promised the fix for this is now live as of v3.0.2.

Thanks,
Simon
